packages/components/tabs/src/tabs.ts

```diff
@@ -110,6 +115,7 @@ export default defineComponent({
 
     const panes: Ref<TabsPaneContext[]> = ref([])
     const currentName = ref(props.modelValue || props.activeName || '0')
+    const scrollPosition = ref(props.scrollPosition || 'auto')
```

Comment:
Since we already have `auto` as default value, I don't think that we really need this fallback.

Reply:
fixed